### DISCUSSION TITLE
Transition docs seem to be wrong, but I might also be misunderstanding them.

### DISCUSSION THREAD

**Discussion by coravacav** [tailwindlabs/headlessui#3195] [Help]
The docs say
> By default, the transition components will render a div element.
> Use the as prop to render the component as a different element or as your own custom component, making sure your custom components [forward refs](https://react.dev/reference/react/forwardRef) so that Headless UI can wire things up correctly.

The problem is if I don't provide an `as`, it seems not to be showing up as a `div` (I've checked the source code to verify this; see [1] below), resulting in the following error when rendering my app.

![image](https://github.com/tailwindlabs/headlessui/assets/40219946/b3ab6ae2-5101-49f5-b847-8c3fbe2be2b7)

I can solve this by passing this to the component instead.

```tsx
as="div"
```

Which resolves the error and lets the app work properly.

I appreciate any help and/or am willing to help out with any changes if necessary

[1]
```tsx
let DEFAULT_TRANSITION_CHILD_TAG = Fragment
```

**Comment by advanced-media** [comment]
if I understand the [changelog of v2](https://github.com/tailwindlabs/headlessui/blob/main/packages/%40headlessui-react/CHANGELOG.md#changed-1) correct, the default is now `Fragment`.

> - Change default tag from div to Fragment on Transition components ([#3110](https://github.com/tailwindlabs/headlessui/pull/3110), [#3147](https://github.com/tailwindlabs/headlessui/pull/3147))

So you are right, the docs of v2 Transition ([rendering as a different element](https://headlessui.com/react/transition#rendering-as-a-different-element)) are wrong.
